### DISCUSSION TITLE
Dealing with an Exponential Data Growth

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#6294] [General]
A blog post from TaxonWorks/Species File Group that feels very timely.

https://github.com/gnames/papers/blob/master/devops-blog-2023/blog.md

**Comment by ewommack** [comment]
I liked how they talk about the accessibility of open source, but then in the end say we all need to do this better. I love the accessibility of our new digital formats, but I still always get stuck on the fact that the printed form is still more archival. How to do we get past that 5 -10 year mark for digital storage without having to rewrite the storage system?
